packages/rancher-logging/generated-changes/patch/values.yaml.patch

```diff
@@ -1,52 +1,94 @@
 --- charts-original/values.yaml
 +++ charts/values.yaml
-@@ -5,7 +5,7 @@
+@@ -5,13 +5,8 @@
  replicaCount: 1
  
  image:
--  repository: ghcr.io/banzaicloud/logging-operator
-+  repository: rancher/mirrored-banzaicloud-logging-operator
-   tag: 3.17.10
+-  # -- Name of the image repository to pull the container image from.
+-  repository: ghcr.io/kube-logging/logging-operator
+-
+-  # -- Image tag override for the default value (chart appVersion).
+-  tag: ""
+-
+-  # -- [Image pull policy](https://kubernetes.io/docs/concepts/containers/images/#updating-images) for updating already existing images on a node.
++  repository: rancher/mirrored-kube-logging-logging-operator
++  tag: 4.4.0
    pullPolicy: IfNotPresent
  
-@@ -42,9 +42,14 @@
+ env: []
+@@ -56,7 +51,7 @@
+ 
+   psp:
+     # -- Must be used with `rbac.enabled` true. If true, creates & uses RBAC resources required in the cluster with [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) enabled.
+-    enabled: true
++    # enabled: false
+ 
+     # -- PSP annotations
+     annotations:
+@@ -70,7 +65,7 @@
+   # specify service account manually
+   # serviceAccountName: custom
+ 
+-monitoring:
++1:
```

Comment:
This patch doesn't look right, but maybe `1:` is the intended values key here?

Reply:
Looks like there is also a monitoring field down below in this values file, potentially a duplicate field mistake while rebasing? not sure

Reply:
Something went weird here, we should have caught this for the initial rebase but didn't